biome.json

```
@@ -12,7 +12,22 @@
         "useExhaustiveDependencies": "off",
         "noUnusedImports": "warn",
         "useJsxKeyInIterable": "off",
-        "useHookAtTopLevel": "error"
+        "useHookAtTopLevel": "error",
+        "useImportExtensions": {
+          "level": "error",
+          "options": {
+            "suggestedExtensions": {
+              "ts": {
+                "module": "js",
+                "component": "jsx"
+              },
+              "tsx": {
+                "module": "js",
```

```suggestion
                "module": "ts",
```
Since we use vite to build here, turns out the correct extension is actually the extension of the file, not what tsc compiles it down to.